server/server.js

```diff
@@ -76,12 +107,25 @@ export default function server(app: express, options: any):express {
     res.json(response);
   });
 
-  app.get("/storedquery", checkConfiguration(), ({ agent }, res) => {
+  routes.get("/storedquery",
+    validationMiddleware,
```

Comment:
Had to inject the validation middleware in specific routes where we wanted validation.  because we can't do it on the entire route without making the readme require validation, which we don't want to do